src/pages/signUp/verification/useVerification.tsx

```diff
@@ -27,17 +29,41 @@ export const useVerification = () => {
       navigate(`/sign-up/detail?email=${email}`);
     }
   };
+
+  const checkCodeExpired = () => {
+    setTimeout(() => {
+      setCodeExpired(true);
+    }, 120000);
```

Comment:
please create a const variable top of your function to have easier access for changing later.